app/views/audits/new.html.erb

```diff
@@ -0,0 +1,63 @@
+<section class="content-header">
+  <% content_for :title, "New - Audits - Inventory - #{current_organization.name}" %>
+  <h1>
+    New Inventory Audit
+    <small>for <%= current_organization.name %></small>
+  </h1>
+  <ol class="breadcrumb">
+    <li><%= link_to(dashboard_path) do %>
+        <i class="fa fa-dashboard"></i> Home
+      <% end %>
+    </li>
+    <li><%= link_to "Inventory Audits", (audits_path) %></li>
+    <li class="active"> New Audit</li>
+  </ol>
+</section>
+
+<!-- Main content -->
+<section class="content">
+
+  <!-- Default box -->
+  <div class="box">
+    <div class="box-header with-border">
+      <p class="box-title">Create an <strong>Audit</strong> recording all the inventory present in a particular storage location.</p>
+    </div>
+
+    <div class="box-body">
+      <p class="help">Enter <code>all</code> the items with their corresponding <code>quantity</code> that are present in the storage location</p>
+      <p class="help">Click on <code>Save Progress</code> button to save the current progress of audit.</p>
+      <p class="help">You can <code>resume</code> the saved audit at a later time again.</p>
+
+      <%= simple_form_for @audit, html: { class: "storage-location-required" } do |f| %>
+
+        <%= render partial: "storage_locations/source", object: f %>
+
+
+        <fieldset style="margin-bottom: 2rem;" class="form-inline">
+          <legend>Items in this audit</legend>
+          <%= f.simple_fields_for :line_items do |item| %>
+            <div id="audit_line_items" class="line-item-fields" data-capture-barcode="true">
+              <%= render 'line_items/line_item_fields', f: item %>
+            </div>
+          <% end %>
+          <div class="row links">
+            <div class="col-xs-12">
+              <%= add_line_item_button f, "#audit_line_items" %>
+            </div>
+          </div>
+        </fieldset>
+
+        <div class="row">
+          <div class="col-xs-12 pull-right">
+            <%= submit_button({ text: 'Confirm Audit', name: 'confirm_audit', icon: 'check-circle', align: 'left', size: 'sm'}, {confirm: 'Are you sure?'}) %>
+            <%= submit_button({ text: 'Save Progress', name: 'save_progress', size: 'md' }) %>
+          </div>
+        </div>
+      <% end %>
```

Comment:
Looks like you should be able to gut the form out of here into a `_form` partial and reference it in both `:new` and `:edit`